package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "micromark",
+  "version": "0.0.1",
+  "description": "markdown parser under @unifiedjs that’s different",
+  "main": "built/lib/index.js",
+  "types": "built/lib/index.d.ts",
+  "module": "lib/index.ts",
+  "scripts": {
+    "build": "rimraf built && tsc",
+    "watch": "tsc --watch",
+    "lint": "tslint -p ./tsconfig.json",
+    "test": "mocha 'test/**/*-test.ts'",
+    "prettier": "prettier --config ./.prettierrc --write \"{test,bin,lib}/**/*.{ts,json}\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/micromark/micromark.git"
+  },
+  "keywords": [
+    "markdown",
+    "lexer",
+    "lexing",
+    "unifiedjs"
+  ],
+  "author": "",
+  "contributors": [],
+  "license": "ISC",
```

Comment:
I think the rest of the unified projects are MIT. That would be my preference at least for a healthy ecosystem